$NetBSD: patch-af,v 1.6 2004/07/28 15:17:42 wiz Exp $

ftp://ftp.kame.net/pub/kame/misc/gnupg-1.2.2-IPv6.diff.gz

--- configure.orig	Tue Dec 23 14:28:22 2003
+++ configure
@@ -13579,7 +13579,7 @@ done
 
 
 
-for ac_func in waitpid wait4 sigaction sigprocmask rand pipe stat
+for ac_func in waitpid wait4 sigaction sigprocmask rand pipe stat getaddrinfo
 do
 as_ac_var=`echo "ac_cv_func_$ac_func" | $as_tr_sh`
 echo "$as_me:$LINENO: checking for $ac_func" >&5
